website/scripts/ros/ros_host_vehicle_cmd.js

```diff
@@ -65,11 +32,78 @@ function subscribeToVehicleCMD()
         }
     });
 }
+/**Accelerator Feedback position from ssc_interface package
+ * Topic: /hardware_interface/throttle_feedback 
+ * Message: automotive_platform_msgs/ThrottleFeedback
+            std_msgs/Header header
+                uint32 seq
+                time stamp
+                string frame_id
+            float32 throttle_pedal
+ */
+function sunscribeToThrottleFeedback()
+{
+    var listener = new ROSLIB.Topic({
+        ros:g_ros,
+        name: T_THROTTLE_FEEDBACK,
+        messageType: M_THROTTLE_FEEDBACK
+    });
+    listener.subscribe(function(message){
+        //Check ROSBridge connection before subscribe a topic
+        IsROSBridgeConnected();
+        if(message!=null && message.throttle_pedal!=null)
+        {
+            let value = message.throttle_pedal;
+            let max, accelerationLimit = '';
+            accelerationLimit = session_hostVehicle.accelerationLimit;
+            max =  accelerationLimit != null && accelerationLimit.length > 0? accelerationLimit : 3;
+            //Accelerator Progress
+            if(g_acceleratorCircle != null)
+            {
+                updateAccerator(g_acceleratorCircle,max,value);
+            }
+        }
+    });
+}
+
+/**brake_feedback from ssc_interface package
+ * Topic: /hardware_interface/brake_feedback 
+ * Message: automotive_platform_msgs/BrakeFeedback
+            std_msgs/Header header
+                uint32 seq
+                time stamp
+                string frame_id
+            float32 brake_pedal
+ */
+function sunscribeToBrakeFeedback()
+{
+    var listener = new ROSLIB.Topic({
+        ros:g_ros,
+        name: T_BRAKE_FEEDBACK,
+        messageType: M_BRAKE_FEEDBACK
+    });
+    listener.subscribe(function(message){
+        //Check ROSBridge connection before subscribe a topic
+        IsROSBridgeConnected();
+        if(message!=null && message.brake_pedal!=null)
+        {
+            let value = message.brake_pedal;
+            let max, brakeLimit = '';
+            brakeLimit = session_hostVehicle.brakeLimit;
+            max =  brakeLimit != null && brakeLimit.length > 0? brakeLimit : 6;
```

Comment:
Dan sorry i don't get why you have to set a brake limit when the value is 0-1?

Reply:
Different vehicle has different brake range. it is supposed to be 0-brakelimit.